test.sh

```diff
@@ -69,7 +72,7 @@ function run_test_docker {
     echo "Running test in docker $name with args $@"
 
     docker run \
-         --rm \
+         "--rm=$DOCKER_RM" \
```

Comment:
The quotes here aren't strictly necessary. I wouldn't add them. `--rm=$DOCKER_RM \` will work